src/frontend/src/icp/components/send/IcSendTokenWizard.svelte

```diff
@@ -107,35 +129,41 @@
 				targetNetworkId: networkId
 			});
 
-			await trackEvent({
-				name: isNetworkIdBitcoin(networkId)
-					? TRACK_COUNT_CONVERT_CKBTC_TO_BTC_SUCCESS
-					: isNetworkIdETH(networkId) && isTokenCkEthLedger($token)
-						? TRACK_COUNT_CONVERT_CKETH_TO_ETH_SUCCESS
-						: isNetworkIdETH(networkId) && isTokenCkErc20Ledger($token)
-							? TRACK_COUNT_CONVERT_CKERC20_TO_ERC20_SUCCESS
-							: TRACK_COUNT_IC_SEND_SUCCESS,
-				metadata: {
-					token: $token.symbol
-				}
-			});
+			await Promise.allSettled([
+				trackTimedEventSuccess(timedEvent),
+				trackEvent({
+					name: isNetworkIdBitcoin(networkId)
+						? TRACK_COUNT_CONVERT_CKBTC_TO_BTC_SUCCESS
+						: isNetworkIdETH(networkId) && isTokenCkEthLedger($token)
+							? TRACK_COUNT_CONVERT_CKETH_TO_ETH_SUCCESS
+							: isNetworkIdETH(networkId) && isTokenCkErc20Ledger($token)
+								? TRACK_COUNT_CONVERT_CKERC20_TO_ERC20_SUCCESS
+								: TRACK_COUNT_IC_SEND_SUCCESS,
```

Comment:
wouldn't it more secure to check if the netwoork is IC as final condition? I know it does not happen, but imagine the network is ETH and it's neither ckETH ledger nor ckERC20 ledger.

Reply:
I would not say "secure" given those are "anonymous analytics hints" but, yeah sure we can totally skip the events if it's an unknown analytics tracked network. I can do that in another PR.